HikePlanner/Models/Trip.cs

```diff
@@ -3,48 +3,81 @@
 using System.Linq;
 using System.Text;
 using Models;
+using System.ComponentModel.DataAnnotations;
 
 namespace Models
 {
     public class Trip
     {
+        private int _distance;
+        private string _creator;
+        private int _activityId;
+
         public Trip()
         { }
 
-        public Trip(int activityId, DateTime startTime, DateTime startDate, DateTime endDate, int distance, string creator)
+        public Trip(int activityId, DateTime startDate, DateTime endDate, int distance, string creator)
         {
             ActivityId = activityId;
-            StartTime = startTime;
             StartDate = startDate;
             EndDate = endDate;
             Distance = distance;
             Creator = creator;
         }
 
-        public Trip(int id, int activityId, DateTime startTime, DateTime startDate, DateTime endDate, int distance, string creator) : this (activityId, startTime, startDate, endDate, distance, creator)
+        public Trip(int id, int activityId, DateTime startDate, DateTime endDate, int distance, string creator) : this (activityId, startDate, endDate, distance, creator)
         {
             Id = id;
         }
-
-        public Trip(int checklistId, int id, int activityId, DateTime startTime, DateTime startDate, DateTime endDate, int distance, string creator) : this (id, activityId, startTime, startDate, endDate, distance, creator)
-        {
-            CheckListId = checklistId;
-        }
-
         public int Id { get; set; }
-        public DateTime StartTime { get; set; }
         public DateTime StartDate { get; set; }
         public DateTime EndDate { get; set; }
+        public DateTime StartTime { get; set; }
```

Comment:
We shouldn't need to separate out the Date and the Time. Both are stored as DateTime object, which counts miliseconds passed since.. 1970 1/1? but anywho, it is capable of storing both date and time. In the front end, we can assemble it into the whole dateTime and then just send it over here

Reply:
Nvm, it's counted with 100-nanosecond units called "ticks" but still, I think we have options of setting time on the date, etc. on client side to facilitate sending it over as a whole dateTime object